CHANGELOG.md

```diff
@@ -1,3 +1,90 @@
+### citus v10.1.0 (June 15, 2021) ###
+
+* Drops support for PostgreSQL 11
+
+* Adds `shard_count` parameter to `create_distributed_table` function
+
+* Adds support for `ALTER DATABASE OWNER`
+
+* Adds support for temporary columnar tables
+
+* Adds `worker_partitioned_table_size`, `worker_partitioned_relation_size`
+  and `worker_partitioned_relation_total_size` UDFs for partitioned size
+  calculation
+
+* `alter_columnar_table_set` enforces columnar table option constraints
+
+* Continues to remove shards after failure in `DropMarkedShards`
+
+* Deprecates the `citus.replication_model` GUC
+
+* Enables `citus.defer_drop_after_shard_move` by default
+
+* Ensures free disk space before moving a shard
+
+* Fetches shard size on the fly for the rebalance monitor
+
+* Ignores old placements when disabling or removing a node
+
+* Implements `improvement_threshold` at shard rebalancer moves
+
+* Improves orphaned shard cleanup logic
+
+* Improves performance of `citus_shards`
+
+* Introduces `citus.local_hostname` GUC for connections to the current node
+
+* Makes sure connection is closed after each shard move
+
+* Makes sure that target node in shard moves is eligible for shard move
+
+* Optimizes partitioned disk size calculation for shard rebalancer
+
+* Prevents connection errors by properly terminating connections
+
+* Prevents inheriting a distributed table
+
+* Prevents users from dropping & truncating known shards
+
+* Pushes down `VALUES` clause as long as not in outer part of a `JOIN`
+
+* Reduces memory usage for multi-row inserts
+
+* Reduces memory usage while rebalancing shards
+
+* Removes length limits around partition names
+
+* Avoids opening extra connections
+
+* Fixes a bug that can cause a crash when DEBUG4 logging is enabled
+
+* Fixes column default coming from a sequence
+
+* Fixes data race in `get_rebalance_progress`
```

Comment:
Does this "data race" exist with the previous implementation of `get_rebalance_progress` @JelteF ? If not we should not include this line here I think because this bug was never seen by users

Reply:
It existed in the previous implementation too.